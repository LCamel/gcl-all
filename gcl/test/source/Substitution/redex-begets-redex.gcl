{: Father = Son + 1 :}
{: Son = 0 :}

{ Father = 1 }
