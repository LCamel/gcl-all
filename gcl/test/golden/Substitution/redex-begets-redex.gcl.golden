0: Father
  Father ===> Son + 1
    1: Son
      Son ===> 0
        